V4-Upgrade-Guide.md

```diff
@@ -48,3 +49,25 @@ To disable caching set `useCache: false` in the options, for example in `execute
 .execute({ destinationName: 'DESTINATION', jwt: 'JWT', useCache: false })
 ```
 
+## Set `iasToXsuaaTokenExchange` to `true` to enable IAS to XSUAA token exchange
+
+**Token exchange from IAS to XSUAA is now disabled by default. Set `iasToXsuaaTokenExchange` to `true` explicitly if token exchange is expected.** 
+
+This change affects the default behaviour of following functions
+
+- `getDestination()`
+- `getAllDestinationsFromDestinationService()`
+- `registerDestination()`
+- `getDestinationFromDestinationService()`
+- `useOrFetchDestination()`
+- `toDestinationNameUrl()`
+- `buildHttpRequest()`
+- `executeHttpRequest()`
+- `executeHttpRequestWithOrigin()`
+
+and following methods of request builder
+
+- `execute()`
+- `executeRaw()`
+- `url()`
+- `build()`
```

Comment:
[req] same goes for build

Reply:
This I however disagree.

https://github.com/SAP/cloud-sdk-js/blob/3c18521964fbc9d30a046b554bf9d7b8015ad45f/packages/odata-common/src/request-builder/request-builder-base.ts#L133

And for any request builder, I can do 

```ts
this.build({
  iasToXsuaaTokenExchange: false,
  destinationName: ''
});
```

Reply:
this is a protected function, mostly called internally. The only .build() method user can call is on the entityBuilder, and that does not accept any arguments. Therefore I will only mention public methods here